vendor/assets/stylesheets/dvl/core/typography.scss

```diff
@@ -9,7 +9,7 @@
 html { font-size: 75%; } // 12px
 @media only screen and (min-width: 320px) { html { font-size: $maxFontSize * 0.8125; } } // 13px
 @media only screen and (min-width: 480px) { html { font-size: $maxFontSize * 0.875; } } // 14px
-@media only screen and (min-width: 640px) { html { font-size: $maxFontSize * 1; } } // 16px
+@media only screen and (min-width: $lapWidth) { html { font-size: $maxFontSize * 1; } } // 16px
```

Comment:
Not really sure if this counts as `$lapWidth`, I think it makes more sense to leave it as a constant.